src/components/profile/profile.spec.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import TestUtils from 'react-dom/test-utils';
+import { shallow } from 'enzyme';
+import TestRenderer from 'react-test-renderer';
+import Profile from './profile.component';
+import { elementsTagTest, rootTagTest } from '../../utils/helpers/tags/tags-specs';
+import Browser from '../../utils/helpers/browser';
+import { ProfileNameStyle, ProfileAvatarStyle } from './profile.style';
+import 'jest-styled-components';
+import ProfileClassicStyle from './profile-classic.style';
+import classicTheme from '../../style/themes/classic';
+
+
+describe('PortraitContainer', () => {
+  let instance;
+  const fillRectFn = jest.fn();
+  const fillTextFn = jest.fn();
+
+  beforeAll(() => {
+    spyOn(Browser, 'getDocument').and.returnValue({
+      createElement: () => {
+        return {
+          getContext: () => {
+            return {
+              font: null,
+              textAlign: null,
+              fillStyle: null,
+              fillRect: fillRectFn,
+              fillText: fillTextFn
+            };
+          },
+          width: 10,
+          height: 10,
+          toDataURL: () => {
+            return 'data:image/png';
+          }
+        };
+      }
+    });
+  });
+
+  describe('render', () => {
+    beforeEach(() => {
+      instance = shallow(
+        <Profile
+          name='Foo'
+          email='foo@bar.com'
+          initials='FB'
+        />
+      );
+    });
+
+    describe('classes', () => {
+      it('returns the correct classes', () => {
+        instance.setProps({ className: 'foo' });
+        expect(instance.hasClass('foo')).toBeTruthy();
+      });
+
+      it('renders the large class if applied', () => {
+        instance = TestRenderer.create(<Profile
+          name='Foo'
+          email='foo@bar.com'
+          initials='FB'
+          large
+        />);
+
+        expect(instance).toMatchSnapshot();
+      });
+    });
+
+    describe('initials', () => {
+      it('calculates the initials when not provided', () => {
+        instance = TestUtils.renderIntoDocument(
+          <Profile
+            name='Foo Bar Baz'
+            email='foo@bar.com'
+          />
+        );
+        expect(instance.initials).toEqual('FBB');
+      });
+    });
+
+    describe('avatar', () => {
+      it('returns the portrait component', () => {
+        expect(ProfileAvatarStyle).toBeTruthy();
```

Comment:
what this test is supposed to do?

Reply:
Profile component uses Portrait component to show the avatar. In this text i wanted to check if the avatar rendered

Reply:
but it will always be truthy